.changeset/silly-coins-approve.md

```diff
@@ -0,0 +1,12 @@
+---
+'@guardian/source-foundations': minor
+---
+
+Adds [web typography presets](https://guardian.github.io/storybooks/?path=/story/source-foundations_typography--presets) to unify typographic language in [designs](https://www.theguardian.design/2a1e5182b/p/01555f-typography-presets/b/830670) and code implementations.
+
+From now on, these presets should be used instead of [the old typography API](https://guardian.github.io/storybooks/?path=/docs/source-foundations_typography-api-deprecated--docs), which this release deprecates.
+
+_Please get in touch with the CSTI team if you need any guidance or assistance with migrating existing uses of the API to the new presets._
```

Comment:
just a suggestion to put the bit about assistance up front, not a blocker, feel free to dismiss if you don't like it

```suggestion
_If you need assistance with migrating existing uses of the API to the new presets, please contact the CSTI team._
```